diff --git a/update_image_palmetto.pl b/update_image_palmetto.pl
index 26c99b2..e925e8a 100755
--- a/update_image_palmetto.pl
+++ b/update_image_palmetto.pl
@@ -89,7 +89,8 @@ run_command("ecc --inject $scratch_dir/hostboot.temp.bin --output $scratch_dir/h
 
 #Create blank binary file for GUARD Data (GUARD) Partition
 run_command("dd if=/dev/zero bs=16K count=1 | tr \"\\000\" \"\\377\" > $scratch_dir/hostboot.temp.bin");
-run_command("ecc --inject $scratch_dir/hostboot.temp.bin --output $scratch_dir/guard.bin.ecc --p8");
+run_command("cp $scratch_dir/hostboot.temp.bin $scratch_dir/guard.bin.ecc");
+#run_command("ecc --inject $scratch_dir/hostboot.temp.bin --output $scratch_dir/guard.bin.ecc --p8");
 
 #Create blank binary file for NVRAM Data (NVRAM) Partition
 run_command("dd if=/dev/zero bs=512K count=1 | tr \"\\000\" \"\\377\" > $scratch_dir/hostboot.temp.bin");
